cockatrice/src/tab_deck_editor.cpp

```diff
@@ -546,9 +552,10 @@ void TabDeckEditor::retranslateUi()
     aSaveDeckToClipboard->setText(tr("Save deck to clip&board"));
     aPrintDeck->setText(tr("&Print deck..."));
 
-    analyzeDeckMenu->setTitle(tr("&Analyze deck online"));
-    aAnalyzeDeckDeckstats->setText("deckstats.net");
-    aAnalyzeDeckTappedout->setText("tappedout.net");
+    analyzeDeckMenu->setTitle(tr("&Send deck to online service"));
+    aExportDeckDecklist->setText(tr("&Create decklist (decklist.org)"));
+    aAnalyzeDeckDeckstats->setText("&Analyze deck (deckstats.net)");
+    aAnalyzeDeckTappedout->setText("&Analyze deck (tappedout.net)");
```

Comment:
In this instance, the `&A` will try to default a shortcut... but both have the same... I'm a fan of not having any of these 4 options have a "windows shortcut" built in

Reply:
Jep... having two &A there won't work. Not even sure if it would work to open a subgroup ("send deck to online service" here) in theory.

But it looks like all shortcuts like this don't work anymore!?
They don't show up for meanymore (used to be highlighted with underlined letters).
Maybe since we have the custom shortcuts in place it's all overwritten?
That's something to check and track in a different issue for sure.

Can you verify that?

Reply:
Hmm, interesting - I missed that when trying to duplicate the rest. I though that for translation all of the strings needed to be prefaced with an &, my mistake, I'll change that real qucik.

Reply:
You could not know that. It used? to make the follow-up letter behind `&` a shortcut for that option:

`Hell&o` would display as "Hello" (o underlined) and would trigger on Ctrl+o